bedrock/mozorg/templates/mozorg/home/home-voices.html

```diff
@@ -359,6 +359,14 @@ <h2 class="primary go">{{ _('Mozilla Developer Network') }}</h2>
   <button id="scroll-prompt">{{ _('Discover more') }}</button>
 </main>
 
+{% if l10n_has_tag('mozorg-home-fundraising-2016') and switch('mozorg-home-fundraising-2016') %}
```

Comment:
Are there many locales still on the old home page? Just curious if adding this is worth the extra complexity. If there are still some locales, it's definitely worth doing.